draft-ietf-mls-protocol.md

```diff
@@ -288,9 +288,11 @@ A              B              C          Directory       Channel
 
 When a participant A wants to establish a group with B and C, it
 first downloads InitKeys for B and C.  It then initializes a group state
-containing only itself and uses the InitKeys to compute Add messages
-to add B and C, in a sequence chosen by A.
-These messages are broadcasted to the Group, and processed in sequence
+containing only itself and uses the InitKeys to compute Welcome and Add messages
+to add B and C, in a sequence chosen by A.  The Welcome messages are
+sent directly to the new members (there is no need to send them to
```

Comment:
Note to self: there is no need to send it to the group, indeed. A nice thing might be for the group to have optional knowledge of who invited someone to the group.